src/empty-state-card.tsx

```diff
@@ -70,24 +70,32 @@ const EmptyStateCard: React.FC<Props> = ({
         image = <img className={classes.image} alt="" src={imageUrl} />;
     }
 
+    let resizedIcon;
+    if (icon) {
+        resizedIcon = React.cloneElement(icon, {size: isTabletOrSmaller ? 64 : 80, color: colors.brand});
+    }
+
+    let smallButton;
+    if (button) {
+        smallButton = React.cloneElement(button, {small: true} as Partial<React.FC<ButtonProps>>);
+    }
```

Comment:
I don't really like this, you're assuming here that you will receive an Icon component with a size prop, but this component accepts anything as icon (it could be, for example, an `<img>`)